_layouts/default.html

```diff
@@ -21,7 +21,7 @@
       {% capture object_id %}{{page.id}}{% endcapture %}
       {% assign object = site.data.resource[object_id] %}
     {% endif %}
-    <div class="wrapper">
+    <div class="wrapper--home">
       <div class="container">
         <main id="main">
           {{content}}
```

Comment:
This changes the background of all pages, not just the homepage. I think we want to keep the search results and item details pages with white backgrounds, yeah?

Reply:
Wouldn't it be a little jarring to have one page different color than the rest? I'd argue for them all being changed.

Reply:
I noticed it because now the background is the same color as the tile background: 
![image](https://user-images.githubusercontent.com/15254041/94200989-51870800-fe89-11ea-8c27-4dd861df53c5.png)

Reply:
@HaSistrunk hmmm that was unintentional. I only meant for the home page to have that background. It wasn't displaying that way for me locally, so let me investigate a little more.